lib/src/base-styles/_nx-card.scss

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-card-container {
+  display: flex;
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
+
+.nx-card {
+  background-color: #fff;
+  border-radius: $nx-border-radius;
+  box-sizing: border-box;
+  padding: $nx-spacing-l $nx-spacing-xxl;
+}
+
+.nx-card__call-out {
+  .nx-icon {
+    font-size: 48px;
+  }
+
+  .nx-icon--large {
+    font-size: 64px;
+  }
+}
+
+.nx-call__call-out-text {
+  @include bold();
+
+  font-size: 32px;
+}
+
+// Vertically oriented cards appear in a row container
+.nx-card-container--row {
+  @include container-horizontal;
+
+  .nx-card {
+    display: flex;
+    flex-direction: column;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    margin: 0 $nx-spacing-md 0 0;
+    text-align: center;
+    min-width: 230px;
+    max-width: 330px;
+  }
+
+  .nx-card-header {
+    @include container-vertical;
+    margin-bottom: $nx-spacing-md;
+  }
+
+  .nx-card__call-out {
+    margin-bottom: $nx-spacing-xs;
+  }
+
+  .nx-card__text {
+    margin-bottom: $nx-spacing-l;
+  }
+
+  .nx-card-footer {
+    border-top: 1px solid #d1d6f0;
+    padding-top: $nx-spacing-l;
+    width: 100%;
+  }
+}
+
+// Horizontally orientated cards appear in a column
+.nx-card-container--column {
+  @include container-vertical;
+
+  flex-direction: column;
+
+  .nx-card {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: auto 1fr auto;
+    grid-template-columns: auto 1fr auto;
+    -ms-grid-rows: auto auto;
+    grid-template-areas:
+      "card-callout card-title card-footer"
+      "card-callout card-text card-footer";
+
+    margin: $nx-spacing-md 0;
+    text-align: left;
+    width: 100%;
+  }
+
+  .nx-card-header {
+    -ms-grid-row: 1;
+    -ms-grid-column: 2;
+    grid-area: card-title;
+
+    @include container-vertical;
+    border-right: 1px solid #d1d6f0;
+    border-left: 1px solid #d1d6f0;
+    margin: 0;
+    padding: 0 $nx-spacing-xxl $nx-spacing-md $nx-spacing-xxl;
```

Comment:
Why padding and not margin?

Reply:
> Why padding and not margin?

Oh because of the borders.  I think the borders need to be moved to the callout and the footer anyway so that they don't appear when those elements aren't present.

Reply:
> Why padding and not margin?

It has to do with how the l/r border is put together.

Reply:
> It has to do with how the l/r border is put together.

See my second comment

Reply:
> > Why padding and not margin?
> 
> Oh because of the borders. I think the borders need to be moved to the callout and the footer anyway so that they don't appear when those elements aren't present.

Exactly, this was changed as a part of: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/b7c8761c4c8ce22a383c0e8e66cead8d0f38827e